assets/src/application/ui/input/dateTime/index.tsx

```diff
@@ -0,0 +1,27 @@
+/* import React, { Suspense } from 'react';
+
+import { LoadingNotice } from '@infraUI/display';
+import { DatepickerProps, TimepickerProps } from './types'; */
+
+// const LazyDatepicker = React.lazy(() => import(/* webpackChunkName: "datepicker" */ './Datepicker'));
+// const LazyTimepicker = React.lazy(() => import(/* webpackChunkName: "timepicker" */ './Timepicker'));
+
+/* export const Datepicker: React.FC<DatepickerProps> = (props) => {
+	return (
+		<Suspense fallback={<LoadingNotice />}>
+			<LazyDatepicker {...props} />
+		</Suspense>
+	);
+};
+
+export const Timepicker: React.FC<TimepickerProps> = (props) => {
+	return (
+		<Suspense fallback={<LoadingNotice />}>
+			<LazyTimepicker {...props} />
+		</Suspense>
+	);
+}; */
```

Comment:
I wanted to lazy load the pickers to see the bundle cost but it didn't help, may be because our production CSS build is not properly set up yet. Leaving it here, may be we can use it